include/imXpadClient.h

```diff
@@ -64,6 +64,7 @@ DEB_CLASS_NAMESPC(DebModCamera, "XpadClient", "Xpad");
 	std::string getErrorMessage() const;
 	std::vector<std::string> getDebugMessages() const;
     int getChar();
+    void sendCustomWait(const std::string& cmd, std::string& value);
```

Comment:
Shouldn't it be better to name it: customSendWait or sendWaitCustom ? (preference for the last)